2.0失败，import还是出现问题

RSSHUB的import 变成 .RSSHUB，或者. 通过了编译器的检查，但运行时也出现问题，应该是包名问题？

~~`from bot import config` 这个我就不造了~~ 哦，要在bot.py里写一个
`from nonebot import scheduler` 可能是因为我个人的问题？ 我的版本高，nonebot==2.0.0a7，nb-cli应该是3.0

```python
12-16 09:13:30 [ERROR] nonebot | Failed to import "RSSHUB"
Traceback (most recent call last):
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "D:\anaconda3\envs\NoneBot-Plugin-Astrologian\Scripts\nb.exe\__main__.py", line 7, in <module>
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\click\core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\click\core.py", line 782, in main
    rv = self.invoke(ctx)
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\click\core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\click\core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\click\core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\nb_cli\__init__.py", line 46, in run
    run_bot(file, app)
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\nb_cli\handlers.py", line 39, in run_bot
    module = importlib.import_module(module_name)
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File ".\bot.py", line 27, in <module>
    nonebot.load_plugins("src/plugins")
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\nonebot\plugin.py", line 889, in load_plugins
    result = context.run(_load_plugin, module_info)
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\nonebot\plugin.py", line 873, in _load_plugin
    module = _load(spec)
  File "<frozen importlib._bootstrap>", line 702, in _load
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "src/plugins\ELF_RSS2\__init__.py", line 9, in <module>
    _sub_plugins |= nonebot.load_plugins(
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\nonebot\plugin.py", line 889, in load_plugins
    result = context.run(_load_plugin, module_info)
> File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\nonebot\plugin.py", line 873, in _load_plugin
    module = _load(spec)
  File "<frozen importlib._bootstrap>", line 702, in _load
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "F:\QQbot\nonebot2_server\src\plugins\ELF_RSS2\plugins\RSSHUB\__init__.py", line 3, in <module>
    from . import del_cache
  File "F:\QQbot\nonebot2_server\src\plugins\ELF_RSS2\plugins\RSSHUB\del_cache.py", line 2, in <module>
    from bot import config
ImportError: cannot import name 'config' from partially initialized module 'bot' (most likely due to a circular import) (.\bot.py)
12-16 09:13:30 [ERROR] nonebot | Failed to import "add_dy"
Traceback (most recent call last):
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "D:\anaconda3\envs\NoneBot-Plugin-Astrologian\Scripts\nb.exe\__main__.py", line 7, in <module>
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\click\core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\click\core.py", line 782, in main
    rv = self.invoke(ctx)
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\click\core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\click\core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\click\core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\nb_cli\__init__.py", line 46, in run
    run_bot(file, app)
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\nb_cli\handlers.py", line 39, in run_bot
    module = importlib.import_module(module_name)
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File ".\bot.py", line 27, in <module>
    nonebot.load_plugins("src/plugins")
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\nonebot\plugin.py", line 889, in load_plugins
    result = context.run(_load_plugin, module_info)
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\nonebot\plugin.py", line 873, in _load_plugin
    module = _load(spec)
  File "<frozen importlib._bootstrap>", line 702, in _load
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "src/plugins\ELF_RSS2\__init__.py", line 9, in <module>
    _sub_plugins |= nonebot.load_plugins(
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\nonebot\plugin.py", line 889, in load_plugins
    result = context.run(_load_plugin, module_info)
> File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\nonebot\plugin.py", line 873, in _load_plugin
    module = _load(spec)
  File "<frozen importlib._bootstrap>", line 702, in _load
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "F:\QQbot\nonebot2_server\src\plugins\ELF_RSS2\plugins\add_dy.py", line 1, in <module>
    from RSSHUB import RSS_class, RWlist, rsstrigger as TR
ModuleNotFoundError: No module named 'RSSHUB'
12-16 09:13:30 [ERROR] nonebot | Failed to import "change_dy"
Traceback (most recent call last):
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "D:\anaconda3\envs\NoneBot-Plugin-Astrologian\Scripts\nb.exe\__main__.py", line 7, in <module>
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\click\core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\click\core.py", line 782, in main
    rv = self.invoke(ctx)
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\click\core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\click\core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\click\core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\nb_cli\__init__.py", line 46, in run
    run_bot(file, app)
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\nb_cli\handlers.py", line 39, in run_bot
    module = importlib.import_module(module_name)
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File ".\bot.py", line 27, in <module>
    nonebot.load_plugins("src/plugins")
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\nonebot\plugin.py", line 889, in load_plugins
    result = context.run(_load_plugin, module_info)
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\nonebot\plugin.py", line 873, in _load_plugin
    module = _load(spec)
  File "<frozen importlib._bootstrap>", line 702, in _load
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "src/plugins\ELF_RSS2\__init__.py", line 9, in <module>
    _sub_plugins |= nonebot.load_plugins(
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\nonebot\plugin.py", line 889, in load_plugins
    result = context.run(_load_plugin, module_info)
> File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\nonebot\plugin.py", line 873, in _load_plugin
    module = _load(spec)
  File "<frozen importlib._bootstrap>", line 702, in _load
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "F:\QQbot\nonebot2_server\src\plugins\ELF_RSS2\plugins\change_dy.py", line 1, in <module>
    from RSSHUB import rsstrigger as TR, RWlist
ModuleNotFoundError: No module named 'RSSHUB'
12-16 09:13:30 [ERROR] nonebot | Failed to import "del_dy"
Traceback (most recent call last):
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "D:\anaconda3\envs\NoneBot-Plugin-Astrologian\Scripts\nb.exe\__main__.py", line 7, in <module>
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\click\core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\click\core.py", line 782, in main
    rv = self.invoke(ctx)
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\click\core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\click\core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\click\core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\nb_cli\__init__.py", line 46, in run
    run_bot(file, app)
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\nb_cli\handlers.py", line 39, in run_bot
    module = importlib.import_module(module_name)
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File ".\bot.py", line 27, in <module>
    nonebot.load_plugins("src/plugins")
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\nonebot\plugin.py", line 889, in load_plugins
    result = context.run(_load_plugin, module_info)
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\nonebot\plugin.py", line 873, in _load_plugin
    module = _load(spec)
  File "<frozen importlib._bootstrap>", line 702, in _load
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "src/plugins\ELF_RSS2\__init__.py", line 9, in <module>
    _sub_plugins |= nonebot.load_plugins(
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\nonebot\plugin.py", line 889, in load_plugins
    result = context.run(_load_plugin, module_info)
> File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\nonebot\plugin.py", line 873, in _load_plugin
    module = _load(spec)
  File "<frozen importlib._bootstrap>", line 702, in _load
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "F:\QQbot\nonebot2_server\src\plugins\ELF_RSS2\plugins\del_dy.py", line 1, in <module>
    from RSSHUB import RWlist
ModuleNotFoundError: No module named 'RSSHUB'
12-16 09:13:30 [ERROR] nonebot | Failed to import "show_all"
Traceback (most recent call last):
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "D:\anaconda3\envs\NoneBot-Plugin-Astrologian\Scripts\nb.exe\__main__.py", line 7, in <module>
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\click\core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\click\core.py", line 782, in main
    rv = self.invoke(ctx)
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\click\core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\click\core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\click\core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\nb_cli\__init__.py", line 46, in run
    run_bot(file, app)
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\nb_cli\handlers.py", line 39, in run_bot
    module = importlib.import_module(module_name)
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File ".\bot.py", line 27, in <module>
    nonebot.load_plugins("src/plugins")
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\nonebot\plugin.py", line 889, in load_plugins
    result = context.run(_load_plugin, module_info)
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\nonebot\plugin.py", line 873, in _load_plugin
    module = _load(spec)
  File "<frozen importlib._bootstrap>", line 702, in _load
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "src/plugins\ELF_RSS2\__init__.py", line 9, in <module>
    _sub_plugins |= nonebot.load_plugins(
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\nonebot\plugin.py", line 889, in load_plugins
    result = context.run(_load_plugin, module_info)
> File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\nonebot\plugin.py", line 873, in _load_plugin
    module = _load(spec)
  File "<frozen importlib._bootstrap>", line 702, in _load
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "F:\QQbot\nonebot2_server\src\plugins\ELF_RSS2\plugins\show_all.py", line 1, in <module>
    from RSSHUB import RWlist
ModuleNotFoundError: No module named 'RSSHUB'
12-16 09:13:30 [ERROR] nonebot | Failed to import "show_dy"
Traceback (most recent call last):
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "D:\anaconda3\envs\NoneBot-Plugin-Astrologian\Scripts\nb.exe\__main__.py", line 7, in <module>
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\click\core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\click\core.py", line 782, in main
    rv = self.invoke(ctx)
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\click\core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\click\core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\click\core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\nb_cli\__init__.py", line 46, in run
    run_bot(file, app)
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\nb_cli\handlers.py", line 39, in run_bot
    module = importlib.import_module(module_name)
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File ".\bot.py", line 27, in <module>
    nonebot.load_plugins("src/plugins")
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\nonebot\plugin.py", line 889, in load_plugins
    result = context.run(_load_plugin, module_info)
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\nonebot\plugin.py", line 873, in _load_plugin
    module = _load(spec)
  File "<frozen importlib._bootstrap>", line 702, in _load
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "src/plugins\ELF_RSS2\__init__.py", line 9, in <module>
    _sub_plugins |= nonebot.load_plugins(
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\nonebot\plugin.py", line 889, in load_plugins
    result = context.run(_load_plugin, module_info)
> File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\nonebot\plugin.py", line 873, in _load_plugin
    module = _load(spec)
  File "<frozen importlib._bootstrap>", line 702, in _load
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "F:\QQbot\nonebot2_server\src\plugins\ELF_RSS2\plugins\show_dy.py", line 1, in <module>
    from RSSHUB import RWlist
ModuleNotFoundError: No module named 'RSSHUB'
12-16 09:13:30 [ERROR] nonebot | Failed to import "start"
Traceback (most recent call last):
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "D:\anaconda3\envs\NoneBot-Plugin-Astrologian\Scripts\nb.exe\__main__.py", line 7, in <module>
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\click\core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\click\core.py", line 782, in main
    rv = self.invoke(ctx)
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\click\core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\click\core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\click\core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\nb_cli\__init__.py", line 46, in run
    run_bot(file, app)
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\nb_cli\handlers.py", line 39, in run_bot
    module = importlib.import_module(module_name)
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File ".\bot.py", line 27, in <module>
    nonebot.load_plugins("src/plugins")
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\nonebot\plugin.py", line 889, in load_plugins
    result = context.run(_load_plugin, module_info)
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\nonebot\plugin.py", line 873, in _load_plugin
    module = _load(spec)
  File "<frozen importlib._bootstrap>", line 702, in _load
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "src/plugins\ELF_RSS2\__init__.py", line 9, in <module>
    _sub_plugins |= nonebot.load_plugins(
  File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\nonebot\plugin.py", line 889, in load_plugins
    result = context.run(_load_plugin, module_info)
> File "d:\anaconda3\envs\nonebot-plugin-astrologian\lib\site-packages\nonebot\plugin.py", line 873, in _load_plugin
    module = _load(spec)
  File "<frozen importlib._bootstrap>", line 702, in _load
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "F:\QQbot\nonebot2_server\src\plugins\ELF_RSS2\plugins\start.py", line 2, in <module>
    from bot import config
ImportError: cannot import name 'config' from partially initialized module 'bot' (most likely due to a circular import) (.\bot.py)
12-16 09:13:30 [INFO] nonebot | Succeeded to import "ELF_RSS2"
12-16 09:13:30 [INFO] nonebot | Running NoneBot...
12-16 09:13:30 [INFO] uvicorn | Started server process [15044]
12-16 09:13:30 [INFO] uvicorn | Waiting for application startup.
12-16 09:13:30 [INFO] uvicorn | Application startup complete.
12-16 09:13:30 [INFO] uvicorn | Uvicorn running on http://127.0.0.1:8080 (Press CTRL+C to quit)


```

--- a/bot.py
+++ b/bot.py
@@ -17,7 +17,10 @@ nonebot.init()
 app = nonebot.get_asgi()
 config = nonebot.get_driver().config
 nonebot.load_builtin_plugins()
+nonebot.load_plugin("nonebot_plugin_apscheduler")
+
 nonebot.load_plugins("src/plugins")
+
 # Modify some config / config depends on loaded configs
 #
 # config = nonebot.get_driver().config

--- a/src/plugins/ELF_RSS2/__init__.py
+++ b/src/plugins/ELF_RSS2/__init__.py
@@ -1,6 +1,10 @@
-import nonebot
-from .config import Config
 from pathlib import Path
+
+import nonebot
+
+from .config import Config
+from .plugins import RSSHUB
+
 global_config = nonebot.get_driver().config
 plugin_config = Config(**global_config.dict())
 # store all subplugins

--- a/src/plugins/ELF_RSS2/config.py
+++ b/src/plugins/ELF_RSS2/config.py
@@ -2,7 +2,6 @@ from pydantic import BaseSettings
 
 
 class Config(BaseSettings):
-
     # plugin custom config
     plugin_setting: str = "default"
 

--- a/src/plugins/ELF_RSS2/plugins/RSSHUB/RSS_class.py
+++ b/src/plugins/ELF_RSS2/plugins/RSSHUB/RSS_class.py
@@ -1,36 +1,40 @@
+import re
 
 from bot import config
-import re
+
+
 class rss:
     # 定义基本属性
-    name = '' # 订阅名
-    url = '' # 订阅地址
-    user_id=[] # 订阅用户（qq） -1 为空
-    group_id=[] # 订阅群组
-    notrsshub=False # 是否非rsshub订阅
-    img_proxy=False
-    sum=20 # 加载条数
-    time=5 #更新频率 分钟/次
-    translation=False # 翻译
-    only_title=False #仅标题
-    only_pic=False #仅图片
+    name = ''  # 订阅名
+    url = ''  # 订阅地址
+    user_id = []  # 订阅用户（qq） -1 为空
+    group_id = []  # 订阅群组
+    notrsshub = False  # 是否非rsshub订阅
+    img_proxy = False
+    sum = 20  # 加载条数
+    time = 5  # 更新频率 分钟/次
+    translation = False  # 翻译
+    only_title = False  # 仅标题
+    only_pic = False  # 仅图片
+
     # 定义构造方法
-    def __init__(self,name:str, url:str, user_id:str, group_id:str,time=5,img_proxy=False,notrsshub=False,translation=False,only_title=False,only_pic=False):
+    def __init__(self, name: str, url: str, user_id: str, group_id: str, time=5, img_proxy=False, notrsshub=False,
+                 translation=False, only_title=False, only_pic=False):
         self.name = name
         self.url = url
-        if user_id!='-1' :
+        if user_id != '-1':
             self.user_id = user_id.split(',')
-        if group_id!='-1' :
+        if group_id != '-1':
             self.group_id = group_id.split(',')
-        self.notrsshub=notrsshub
-        self.time=time
-        self.img_proxy=img_proxy
-        self.translation=translation
-        self.only_title=only_title
-        self.only_pic=only_pic
+        self.notrsshub = notrsshub
+        self.time = time
+        self.img_proxy = img_proxy
+        self.translation = translation
+        self.only_title = only_title
+        self.only_pic = only_pic
 
-    def geturl(self)->str:
-        if re.match(u'[hH][tT]{2}[pP][sS]{0,}://', self.url, flags=0) :
+    def geturl(self) -> str:
+        if re.match(u'[hH][tT]{2}[pP][sS]{0,}://', self.url, flags=0):
             return self.url
         else:
-            return config.rsshub+self.url
+            return config.rsshub + self.url

--- a/src/plugins/ELF_RSS2/plugins/RSSHUB/RWlist.py
+++ b/src/plugins/ELF_RSS2/plugins/RSSHUB/RWlist.py
@@ -1,28 +1,33 @@
-
-import json
 import codecs
+import json
+import os
 import os.path
-from RSSHUB import RSS_class
+from pathlib import Path
+
+from . import RSS_class
 
 # 存储目录
-file_path = './data/'
+file_path = Path.cwd() / 'data'
+
+
 # 读取记录
 def readRss() -> list:
-    rss_list=[]
-    with codecs.open(file_path+"rss.json", 'r','utf-8') as load_f:
+    rss_list = []
+    with codecs.open(str(file_path / "rss.json"), 'r', 'utf-8') as load_f:
         rss_list_json = json.load(load_f)
         for rss_one in rss_list_json:
-            tmp_rss= RSS_class.rss('', '', '', '')
-            tmp_rss.__dict__=json.loads(rss_one)
+            tmp_rss = RSS_class.rss('', '', '', '')
+            tmp_rss.__dict__ = json.loads(rss_one)
             rss_list.append(tmp_rss)
     return rss_list
+
+
 # 写入记录
-def writeRss(rss_list:list) :
-    rss_json=[]
+def writeRss(rss_list: list):
+    rss_json = []
     for rss_one in rss_list:
-        rss_json.append(json.dumps(rss_one.__dict__,sort_keys=True, indent=4, ensure_ascii=False))
+        rss_json.append(json.dumps(rss_one.__dict__, sort_keys=True, indent=4, ensure_ascii=False))
     if not os.path.isdir(file_path):
         os.makedirs(file_path)
-    with codecs.open(file_path+"rss.json", "w",'utf-8') as dump_f:
+    with codecs.open(str(file_path / "rss.json"), "w", 'utf-8') as dump_f:
         dump_f.write(json.dumps(rss_json, sort_keys=True, indent=4, ensure_ascii=False))
-

--- a/src/plugins/ELF_RSS2/plugins/RSSHUB/__init__.py
+++ b/src/plugins/ELF_RSS2/plugins/RSSHUB/__init__.py
@@ -1,7 +1,11 @@
-__package__='RSSHUB'
+# __package__ = 'RSSHUB'
+
+from . import RSS_class
+from . import RWlist
 from . import del_cache
 from . import rss_baidutrans
 from . import rsshub
-from . import RSS_class
-from . import RWlist
 from . import rsstrigger
+# from nonebot import require
+# scheduler = require("nonebot_plugin_apscheduler")
+# RSSHUB = require("RSSHUB")

--- a/src/plugins/ELF_RSS2/plugins/RSSHUB/del_cache.py
+++ b/src/plugins/ELF_RSS2/plugins/RSSHUB/del_cache.py
@@ -1,11 +1,19 @@
-import nonebot
-from bot import config
 import shutil
-from apscheduler.triggers.interval import IntervalTrigger # 间隔触发器
-from nonebot import scheduler, logger
+from pathlib import Path
 
+import nonebot
+from apscheduler.triggers.interval import IntervalTrigger  # 间隔触发器
+from nonebot import logger
+from nonebot import require
+
+from bot import config
+
+scheduler = require("nonebot_plugin_apscheduler").scheduler
 # 图片存储目录
-file_path = './data/imgs'
+file_path = Path.cwd() / 'data' / 'imgs'
+
+
+# file_path = './data/imgs'
 
 async def del_img(int):
     bot, = nonebot.get_bots().values()
@@ -16,14 +24,16 @@ async def del_img(int):
         logger.error(e)
         await bot.send_msg(message_type='private', user_id=str(list(config.superusers)[0]), message='图片缓存删除失败！')
 
+
 def delcache_trigger():
     # 制作一个“time分钟/次”触发器
     trigger = IntervalTrigger(
         days=config.delcache,
-        #minutes=1,
+        # minutes=1,
         jitter=10
     )
     # 添加任务
+
     scheduler.add_job(
         func=del_img,  # 要添加任务的函数，不要带参数
         trigger=trigger,  # 触发器

--- a/src/plugins/ELF_RSS2/plugins/RSSHUB/rss_baidutrans.py
+++ b/src/plugins/ELF_RSS2/plugins/RSSHUB/rss_baidutrans.py
@@ -15,11 +15,11 @@
 
 '''
 
-import http.client
 import hashlib
+import http.client
 import json
-import urllib
 import random
+import urllib
 
 from nonebot import logger
 

--- a/src/plugins/ELF_RSS2/plugins/RSSHUB/rsshub.py
+++ b/src/plugins/ELF_RSS2/plugins/RSSHUB/rsshub.py
@@ -1,57 +1,62 @@
 # -*- coding: UTF-8 -*-
 
-from io import BytesIO
-import unicodedata
-import feedparser
-import json
-import codecs
 import asyncio
-from PIL import Image
+import codecs
+import difflib
+import json
+import os.path
+import re
+import time
+import uuid
+from io import BytesIO
+from pathlib import Path
+
+import emoji
+import feedparser
 import httpx
 import nonebot
-from pyquery import PyQuery as pq
-import re
-import os.path
 import requests
-import uuid
-import time
-from bot import config
-import difflib
-from nonebot.log import logger
-from RSSHUB import RSS_class
+import unicodedata
+from PIL import Image
 from googletrans import Translator
-import emoji
+from nonebot.log import logger
+from pyquery import PyQuery as pq
 from retrying import retry
 
+from bot import config
+from . import RSS_class
+
 # 存储目录
-file_path = './data/'
-#代理
+file_path = Path.cwd() / 'data'
+# 代理
 proxy = config.rss_proxy
 proxies = {
     'http': 'http://' + str(proxy),
     'https': 'https://' + str(proxy),
 }
-status_code=[200,301,302]
+status_code = [200, 301, 302]
 # 去掉烦人的 returning true from eof_received() has no effect when using ssl httpx 警告
 asyncio.log.logger.setLevel(40)
+
+
 @retry
-async def getRSS(rss: RSS_class.rss)->list:# 链接，订阅名
-    #设置全局超时 以解决 feedparser.parse 遇到 bad url 时卡住
-    #socket.setdefaulttimeout(5000)
+async def getRSS(rss: RSS_class.rss) -> list:  # 链接，订阅名
+    # 设置全局超时 以解决 feedparser.parse 遇到 bad url 时卡住
+    # socket.setdefaulttimeout(5000)
     if rss.img_proxy:
         Proxy = {
-            "all":"http://" + str(proxy)
+            "all": "http://" + str(proxy)
         }
     else:
         Proxy = {}
     try:
         # 检查是否存在rss记录
-        if os.path.isfile(file_path + rss.name + '.json'):
+        if os.path.isfile(file_path / (rss.name + '.json')):
             # 异步获取 xml
             async with httpx.AsyncClient(proxies=Proxy) as client:
-                d=""
+                d = ""
                 try:
-                    r = await client.get(rss.geturl(),timeout=30)
+                    r = await client.get(rss.geturl(), timeout=30)
                     d = feedparser.parse(r.content)
                 except BaseException as e:
                     logger.error(e)
@@ -67,7 +72,7 @@ async def getRSS(rss: RSS_class.rss)->list:# 链接，订阅名
                                 if r.status_code in status_code:
                                     d = feedparser.parse(r.content)
                                     logger.info(rsshub_url + ' 抓取成功！')
-                                    break;
+                                    break
 
                 change = checkUpdate(d, readRss(rss.name))  # 检查更新
                 if len(change) > 0:
@@ -84,7 +89,8 @@ async def getRSS(rss: RSS_class.rss)->list:# 链接，订阅名
                             Similarity = difflib.SequenceMatcher(None, text, item['title'])
                             if Similarity.quick_ratio() <= 0.1:  # 标题正文相似度
                                 msg = msg + '标题：' + item['title'] + '\n'
-                            msg = msg + '内容：' + await checkstr(item['summary'], rss.img_proxy, rss.translation, rss.only_pic) + '\n'
+                            msg = msg + '内容：' + await checkstr(item['summary'], rss.img_proxy, rss.translation,
+                                                               rss.only_pic) + '\n'
                         else:
                             msg = msg + '标题：' + item['title'] + '\n'
                         str_link = re.sub('member_illust.php\?mode=medium&illust_id=', 'i/', item['link'])
@@ -116,10 +122,11 @@ async def getRSS(rss: RSS_class.rss)->list:# 链接，订阅名
                     logger.error('出现异常，获取 ' + rss.name + ' 订阅xml失败！！！请检查订阅地址是否可用！  E:' + str(e))
                 return []
     except BaseException as e:
-        logger.error(rss.name + ' 抓取失败，请检查订阅地址是否正确！ E:'+str(e))
+        logger.error(rss.name + ' 抓取失败，请检查订阅地址是否正确！ E:' + str(e))
         return []
 
-async def sendMsg(rss,msg,bot):
+
+async def sendMsg(rss, msg, bot):
     try:
         if len(msg) <= 0:
             return
@@ -128,31 +135,32 @@ async def sendMsg(rss,msg,bot):
                 try:
                     await bot.send_msg(message_type='private', user_id=id, message=str(msg))
                 except Exception as e:
-                    logger.error('QQ号'+id+'不合法或者不是好友 E:' + str(e))
+                    logger.error('QQ号' + id + '不合法或者不是好友 E:' + str(e))
 
         if rss.group_id:
             for id in rss.group_id:
                 try:
                     await bot.send_msg(message_type='group', group_id=id, message=str(msg))
                 except Exception as e:
-                    logger.info('群号'+id+'不合法或者未加群 E:' + str(e))
+                    logger.info('群号' + id + '不合法或者未加群 E:' + str(e))
 
     except Exception as e:
-        logger.info('发生错误 消息发送失败 E:'+str(e))
+        logger.info('发生错误 消息发送失败 E:' + str(e))
+
 
 # 下载图片
 @retry
-async def dowimg(url:str,img_proxy:bool)->str:
+async def dowimg(url: str, img_proxy: bool) -> str:
     try:
-        img_path = file_path + 'imgs' + os.sep
+        img_path = file_path / 'imgs' / os.sep
         if not os.path.isdir(img_path):
-            logger.info(img_path + '文件夹不存在，已重新创建')
+            logger.info(str(img_path) + '文件夹不存在，已重新创建')
             os.makedirs(img_path)  # 创建目录
         file_suffix = os.path.splitext(url)  # 返回列表[路径/文件名，文件后缀]
         name = str(uuid.uuid4())
         if img_proxy:
             Proxy = httpx.Proxy(
-                url="http://"+proxy,
+                url="http://" + proxy,
                 mode="TUNNEL_ONLY"  # May be "TUNNEL_ONLY" or "FORWARD_ONLY". Defaults to "DEFAULT".
             )
         else:
@@ -173,14 +181,13 @@ async def dowimg(url:str,img_proxy:bool)->str:
 
                     # 使用第三方反代服务器
                     url = re.sub('i.pximg.net', config.pixiv_proxy, url)
-                    pic = await client.get(url, headers=headers ,timeout=100.0)
+                    pic = await client.get(url, headers=headers, timeout=100.0)
                 else:
                     pic = await client.get(url)
 
-
                 # 大小控制，图片压缩
-                if (float(len(pic.content)/1024) > float(config.zip_size)):
-                    filename = await zipPic(pic.content,name)
+                if (float(len(pic.content) / 1024) > float(config.zip_size)):
+                    filename = await zipPic(pic.content, name)
                 else:
                     if len(file_suffix[1]) > 0:
                         filename = name + file_suffix[1]
@@ -190,16 +197,16 @@ async def dowimg(url:str,img_proxy:bool)->str:
                         filename = name + '.png'
                     else:
                         filename = name + '.jpg'
-                    with codecs.open(img_path + filename, "wb") as dump_f:
+                    with codecs.open(str(img_path / filename), "wb") as dump_f:
                         dump_f.write(pic.content)
 
                 if config.islinux:
-                    imgs_name = img_path + filename
+                    imgs_name = img_path / filename
                     if len(imgs_name) > 0:
                         imgs_name = os.getcwd() + re.sub(r'\./|\\', r'/', imgs_name)
                     return imgs_name
                 else:
-                    imgs_name = img_path + filename
+                    imgs_name = img_path / filename
                     if len(imgs_name) > 0:
                         imgs_name = os.getcwd() + re.sub('/', r'\\', imgs_name)
                         imgs_name = re.sub(r'\\', r'\\\\', imgs_name)
@@ -212,31 +219,31 @@ async def dowimg(url:str,img_proxy:bool)->str:
         logger.error('图片下载失败 1 E:' + str(e))
         return ''
 
-async def zipPic(content,name):
-    img_path = file_path + 'imgs' + os.sep
+
+async def zipPic(content, name):
+    img_path = file_path / 'imgs' / os.sep
     # 打开一个jpg/png图像文件，注意是当前路径:
     im = Image.open(BytesIO(content))
     # 获得图像尺寸:
     w, h = im.size
     logger.info('Original image size: %sx%s' % (w, h))
     # 算出缩小比
-    Proportion = int(len(content)/(float(config.zip_size) * 1024))
-    logger.info('算出的缩小比:'+str(Proportion))
+    Proportion = int(len(content) / (float(config.zip_size) * 1024))
+    logger.info('算出的缩小比:' + str(Proportion))
     # 缩放
     im.thumbnail((w // Proportion, h // Proportion))
     logger.info('Resize image to: %sx%s' % (w // Proportion, h // Proportion))
     # 把缩放后的图像用jpeg格式保存:
     try:
-        im.save(img_path + name + '.jpg', 'jpeg')
+        im.save(img_path / name + '.jpg', 'jpeg')
         return name + '.jpg'
     except Exception:
-        im.save(img_path + name + '.png', 'png')
+        im.save(img_path / name + '.png', 'png')
         return name + '.png'
 
 
-#处理正文
-async def checkstr(rss_str:str,img_proxy:bool,translation:bool,only_pic:bool)->str:
-
+# 处理正文
+async def checkstr(rss_str: str, img_proxy: bool, translation: bool, only_pic: bool) -> str:
     # 去掉换行
     rss_str = re.sub('\n', '', rss_str)
 
@@ -263,12 +270,12 @@ async def checkstr(rss_str:str,img_proxy:bool,translation:bool,only_pic:bool)->s
     rss_str = re.sub('<iframe.+?\"/>', '', rss_str)
     rss_str = re.sub('<i.+?\">|<i>|</i>', '', rss_str)
     rss_str = re.sub('<code>|</code>|<ul>|</ul>', '', rss_str)
-    #解决 issue #3
+    # 解决 issue #3
     rss_str = re.sub('<dd.+?\">|<dd>|</dd>', '', rss_str)
     rss_str = re.sub('<dl.+?\">|<dl>|</dl>', '', rss_str)
     rss_str = re.sub('<dt.+?\">|<dt>|</dt>', '', rss_str)
 
-    rss_str_tl = rss_str # 翻译用副本
+    rss_str_tl = rss_str  # 翻译用副本
     # <a> 标签处理
     doc_a = doc_rss('a')
     for a in doc_a.items():
@@ -295,8 +302,6 @@ async def checkstr(rss_str:str,img_proxy:bool,translation:bool,only_pic:bool)->s
         else:
             rss_str = re.sub(re.escape(str(img)), r'\n图片走丢啦！\n', rss_str, re.S)
 
-
-
     # 处理视频
     doc_video = doc_rss('video')
     for video in doc_video.items():
@@ -304,12 +309,9 @@ async def checkstr(rss_str:str,img_proxy:bool,translation:bool,only_pic:bool)->s
         img_path = await dowimg(video.attr("poster"), img_proxy)
         if len(img_path) > 0:
             rss_str = re.sub(re.escape(str(video)), r'视频封面：[CQ:image,file=file:///' + str(img_path) + ']',
-                                 rss_str)
+                             rss_str)
         else:
             rss_str = re.sub(re.escape(str(video)), r'视频封面：\n图片走丢啦！\n', rss_str)
-
-
-
 
     # 翻译
     text = ''
@@ -327,12 +329,12 @@ async def checkstr(rss_str:str,img_proxy:bool,translation:bool,only_pic:bool)->s
                 text = re.sub(r':[A-Za-z_]*:', ' ', text)
                 text = '\n翻译(BaiduAPI)：\n' + rss_baidutrans.baidu_translate(re.escape(text))
             else:
-               text = '\n翻译：\n' + translator.translate(re.escape(text), dest='zh-CN').text
+                text = '\n翻译：\n' + translator.translate(re.escape(text), dest='zh-CN').text
             text = re.sub(r'\\', '', text)
             text = re.sub(r'百度翻译', '\n', text)
         except Exception as e:
-            text = '\n翻译失败！'+str(e)+'\n'
-    return rss_str+text
+            text = '\n翻译失败！' + str(e) + '\n'
+    return rss_str + text
 
 
 # 检查更新
@@ -344,22 +346,22 @@ def checkUpdate(new, old) -> list:
         return []
     b = old['entries']
 
-    c = [];
+    c = []
     # 防止 rss 超过设置的缓存条数
-    if len(a)>= config.limt:
-        LIMT=len(a) + config.limt
+    if len(a) >= config.limt:
+        LIMT = len(a) + config.limt
     else:
-        LIMT=config.limt
+        LIMT = config.limt
 
     for i in a:
-        count = 0;
+        count = 0
         for j in b:
             if i['id'] == j['id']:
                 count = 1
         if count == 0:
             c.insert(0, i)
     for i in c:
-        count = 0;
+        count = 0
         for j in b:
             if i['id'] == j['id']:
                 count = 1
@@ -370,7 +372,7 @@ def checkUpdate(new, old) -> list:
 
 # 读取记录
 def readRss(name):
-    with codecs.open(file_path + name + ".json", 'r', 'utf-8') as load_f:
+    with codecs.open(file_path / (name + ".json"), 'r', 'utf-8') as load_f:
         load_dict = json.load(load_f)
     return load_dict
 
@@ -388,7 +390,7 @@ def writeRss(new, name):
 
         for tmp in change:
             old['entries'].insert(0, tmp)
-        count = 0;
+        count = 0
         for i in old['entries']:
             count = count + 1
             if count > LIMT:
@@ -398,5 +400,5 @@ def writeRss(new, name):
 
     if not os.path.isdir(file_path):
         os.makedirs(file_path)
-    with codecs.open(file_path + name + ".json", "w", 'utf-8') as dump_f:
+    with codecs.open(file_path / (name + ".json"), "w", 'utf-8') as dump_f:
         dump_f.write(json.dumps(old, sort_keys=True, indent=4, ensure_ascii=False))

--- a/src/plugins/ELF_RSS2/plugins/RSSHUB/rsstrigger.py
+++ b/src/plugins/ELF_RSS2/plugins/RSSHUB/rsstrigger.py
@@ -1,8 +1,13 @@
+from nonebot import require
+
+from . import RSS_class, rsshub
+
+scheduler = require("nonebot_plugin_apscheduler").scheduler
 
 from nonebot.log import logger
-from RSSHUB import rsshub,RSS_class
-from apscheduler.triggers.interval import IntervalTrigger # 间隔触发器
-from nonebot import scheduler
+
+# from RSSHUB import rsshub,RSS_class
+from apscheduler.triggers.interval import IntervalTrigger  # 间隔触发器
 
 
 # 检测某个rss更新 #任务体
@@ -10,7 +15,8 @@ async def check_update(rss: RSS_class.rss):
     logger.info('检查 ' + rss.name + ' 更新')
     await rsshub.getRSS(rss)
 
-def rss_trigger(times:int, rss: RSS_class.rss):
+
+def rss_trigger(times: int, rss: RSS_class.rss):
     # 制作一个“time分钟/次”触发器
     trigger = IntervalTrigger(
         minutes=times,

--- a/src/plugins/ELF_RSS2/plugins/add_dy.py
+++ b/src/plugins/ELF_RSS2/plugins/add_dy.py
@@ -1,12 +1,14 @@
-from RSSHUB import RSS_class, RWlist, rsstrigger as TR
-from nonebot.log import logger
-from bot import config
 import re
-from nonebot import on_command,permission
-from nonebot.rule import to_me
-from nonebot.adapters.cqhttp import Bot, Event
 
-RssAdd = on_command('add',aliases={'订阅','rssadd','dy'}, rule=to_me(), priority=5,permission=permission.SUPERUSER)
+from RSSHUB import RSS_class, RWlist, rsstrigger as TR
+from nonebot import on_command, permission
+from nonebot.adapters.cqhttp import Bot, Event
+from nonebot.log import logger
+from nonebot.rule import to_me
+
+from bot import config
+
+RssAdd = on_command('add', aliases={'订阅', 'rssadd', 'dy'}, rule=to_me(), priority=5, permission=permission.SUPERUSER)
 
 
 @RssAdd.handle()
@@ -16,7 +18,8 @@ async def handle_first_receive(bot: Bot, event: Event, state: dict):
         state["RssAdd"] = args  # 如果用户发送了参数则直接赋值
 
 
-@RssAdd.got("RssAdd", prompt="要订阅的信息不能为空呢，请重新输入\n输入样例：\ntest /twitter/user/xx 11,11 -1 5 0 1 \n订阅名 订阅地址 qq(,分隔，为空-1) 群号(,分隔，为空-1) 更新时间(分钟，可选) 1/0(代理，可选)")
+@RssAdd.got("RssAdd",
+            prompt="要订阅的信息不能为空呢，请重新输入\n输入样例：\ntest /twitter/user/xx 11,11 -1 5 0 1 \n订阅名 订阅地址 qq(,分隔，为空-1) 群号(,分隔，为空-1) 更新时间(分钟，可选) 1/0(代理，可选)")
 async def handle_RssAdd(bot: Bot, event: Event, state: dict):
     rss_dy_link = state["RssAdd"]
     user_id = event.user_id
@@ -30,13 +33,12 @@ async def handle_RssAdd(bot: Bot, event: Event, state: dict):
     # else:
     #     rss_dy_link = await RssAdd.reject('要订阅的信息不能为空呢，请重新输入\n输入样例：\ntest /twitter/user/xx 11,11 -1 5 1 0 \n订阅名 订阅地址 qq(,分隔，为空-1) 群号(,分隔，为空-1) 更新时间(分钟，可选) 1/0(代理，可选) 1/0(翻译,可选) 1/0(仅标题,可选) 1/0(仅图片,可选)')
 
-
     # 获取、处理信息
     dy = rss_dy_link.split(' ')
     try:
         name = dy[0]
-        name=re.sub(r'\?|\*|\:|\"|\<|\>|\\|/|\|', '_', name)
-        if name=='rss':
+        name = re.sub(r'\?|\*|\:|\"|\<|\>|\\|/|\|', '_', name)
+        if name == 'rss':
             name = 'rss_'
         try:
             url = dy[1]
@@ -55,7 +57,7 @@ async def handle_RssAdd(bot: Bot, event: Event, state: dict):
                 elif old.name == name:
                     flag = 3
         except BaseException as e:
-            logger.info("E :"+str(e))
+            logger.info("E :" + str(e))
         if group_id:
             if flag == 0 and url:
                 if len(dy) > 2:
@@ -95,7 +97,7 @@ async def handle_RssAdd(bot: Bot, event: Event, state: dict):
         elif user_id and flag == 0:
             user_id = dy[2]
             group_id = dy[3]
-            if len(dy) > 4 and int(dy[4])>0:
+            if len(dy) > 4 and int(dy[4]) > 0:
                 times = int(dy[4])
             else:
                 times = 5
@@ -124,7 +126,8 @@ async def handle_RssAdd(bot: Bot, event: Event, state: dict):
             logger.info('添加' + name + '失败，已存在')
             await RssAdd.send('订阅名或订阅链接已经存在！')
             return
-        rss = RSS_class.rss(name, url, str(user_id), str(group_id), times, proxy, notrsshub, translation, only_title, only_pic)
+        rss = RSS_class.rss(name, url, str(user_id), str(group_id), times, proxy, notrsshub, translation, only_title,
+                            only_pic)
         # 写入订阅配置文件
         try:
             list_rss.append(rss)
@@ -142,7 +145,3 @@ async def handle_RssAdd(bot: Bot, event: Event, state: dict):
     except BaseException as e:
         logger.info(e)
         await RssAdd.send('参数不对哟！\n关于插件：http://ii1.fun/7byIVb')
-
-
-
-

--- a/src/plugins/ELF_RSS2/plugins/change_dy.py
+++ b/src/plugins/ELF_RSS2/plugins/change_dy.py
@@ -1,14 +1,17 @@
 from RSSHUB import rsstrigger as TR, RWlist
-from nonebot.log import logger
-from nonebot import scheduler, permission
-from nonebot import on_command
-from nonebot.rule import to_me
+# from nonebot import scheduler,
+from nonebot import on_command, permission, require
 from nonebot.adapters.cqhttp import Bot, Event
+from nonebot.log import logger
+from nonebot.rule import to_me
 
+scheduler = require("nonebot_plugin_apscheduler").scheduler
+# from pathlib impo
 # 存储目录
-file_path = './data/'
+# file_path = './data/'
 
-RssChange = on_command('change', aliases={'changedy','moddy'}, rule=to_me(), priority=5,permission=permission.SUPERUSER)
+RssChange = on_command('change', aliases={'changedy', 'moddy'}, rule=to_me(), priority=5,
+                       permission=permission.SUPERUSER)
 
 
 @RssChange.handle()
@@ -18,7 +21,8 @@ async def handle_first_receive(bot: Bot, event: Event, state: dict):
         state["RssChange"] = args  # 如果用户发送了参数则直接赋值
 
 
-@RssChange.got("RssChange", prompt='输入要修改的订阅的 \n订阅名 修改项=,属性 \n如:\n 订阅 dyqq=,xx dsf=0\n对应参数:地址-url，QQ-dyqq，群-dyqun，更新频率-uptime，代理-proxy,翻译-tl，仅title-ot，仅图片-op\n注：\n代理、翻译、仅title属性值为1/0\nqq、群号前加英文逗号表示追加')
+@RssChange.got("RssChange",
+               prompt='输入要修改的订阅的 \n订阅名 修改项=,属性 \n如:\n 订阅 dyqq=,xx dsf=0\n对应参数:地址-url，QQ-dyqq，群-dyqun，更新频率-uptime，代理-proxy,翻译-tl，仅title-ot，仅图片-op\n注：\n代理、翻译、仅title属性值为1/0\nqq、群号前加英文逗号表示追加')
 async def handle_RssAdd(bot: Bot, event: Event, state: dict):
     change_info = state["RssChange"]
     # user_id = event.user_id
@@ -26,7 +30,7 @@ async def handle_RssAdd(bot: Bot, event: Event, state: dict):
     #     group_id = event.group_id
     # except:
     #     group_id = None
-    
+
     flag = 0
     try:
         list_rss = RWlist.readRss()
@@ -35,8 +39,8 @@ async def handle_RssAdd(bot: Bot, event: Event, state: dict):
         name = list_info[0]  # 取订阅名
         list_info.pop(0)  # 从列表删除
         for rss_ in list_rss:
-            if rss_.name == name :
-                rss_a=rss_
+            if rss_.name == name:
+                rss_a = rss_
                 flag = flag + 1
         if flag <= 0:
             await RssChange.send('订阅 ' + name + ' 不存在！')
@@ -44,7 +48,7 @@ async def handle_RssAdd(bot: Bot, event: Event, state: dict):
             try:
                 rss_tmp = rss_a
                 for info in list_info:
-                    info_this = info.split('=',1)
+                    info_this = info.split('=', 1)
                     if info_this[0] == 'url':
                         rss_tmp.url = info_this[1]
                     if info_this[0] == 'dyqq':
@@ -53,7 +57,7 @@ async def handle_RssAdd(bot: Bot, event: Event, state: dict):
                             list_user_id.pop(0)
                             rss_tmp.user_id = rss_tmp.user_id + list_user_id
                         else:
-                            if info_this[1]!='-1':
+                            if info_this[1] != '-1':
                                 rss_tmp.user_id = list_user_id
                             else:
                                 rss_tmp.user_id = []
@@ -95,7 +99,3 @@ async def handle_RssAdd(bot: Bot, event: Event, state: dict):
                 logger.error(e)
     except:
         await RssChange.send('你还没有任何订阅！\n关于插件：http://ii1.fun/7byIVb')
-
-
-
-

--- a/src/plugins/ELF_RSS2/plugins/del_dy.py
+++ b/src/plugins/ELF_RSS2/plugins/del_dy.py
@@ -1,14 +1,18 @@
-from RSSHUB import RWlist
-from nonebot.log import logger
-from nonebot import scheduler, permission
-from nonebot import on_command
-from nonebot.rule import to_me
-from nonebot.adapters.cqhttp import Bot, Event
-import os, re
-# 存储目录
-file_path = './data/'
+import os
+import re
+from pathlib import Path
 
-Rssdel = on_command('deldy', aliases={'delrss','rssdel'}, rule=to_me(), priority=5,permission=permission.SUPERUSER)
+from RSSHUB import RWlist
+from nonebot import on_command, require, permission
+from nonebot.adapters.cqhttp import Bot, Event
+from nonebot.log import logger
+from nonebot.rule import to_me
+
+scheduler = require("nonebot_plugin_apscheduler").scheduler
+# 存储目录
+file_path = Path.cwd() / 'data'
+
+Rssdel = on_command('deldy', aliases={'delrss', 'rssdel'}, rule=to_me(), priority=5, permission=permission.SUPERUSER)
 
 
 @Rssdel.handle()
@@ -26,14 +30,15 @@ async def handle_RssAdd(bot: Bot, event: Event, state: dict):
         group_id = event.group_id
     except:
         group_id = None
-        
+
     # 获取、处理信息
     flag = 0
     try:
         list_rss = RWlist.readRss()
         if group_id:
             for rss_ in list_rss:
-                if (rss_.name == rss_name and str(group_id) in str(rss_.group_id)) or (rss_.url == rss_name and str(group_id) in str(rss_.group_id)):
+                if (rss_.name == rss_name and str(group_id) in str(rss_.group_id)) or (
+                        rss_.url == rss_name and str(group_id) in str(rss_.group_id)):
                     rss_tmp = rss_
                     if rss_tmp.group_id[0] == str(group_id):
                         rss_tmp.group_id.pop(0)
@@ -44,7 +49,7 @@ async def handle_RssAdd(bot: Bot, event: Event, state: dict):
                         list_rss.remove(rss_)
                         scheduler.remove_job(rss_.name)
                         try:
-                            os.remove(file_path+rss_.name+".json")
+                            os.remove(file_path / (rss_.name + ".json"))
                         except BaseException as e:
                             logger.info(e)
                         RWlist.writeRss(list_rss)
@@ -58,7 +63,7 @@ async def handle_RssAdd(bot: Bot, event: Event, state: dict):
                     list_rss.remove(rss_)
                     scheduler.remove_job(rss_.name)
                     try:
-                        os.remove(file_path+rss_.name+".json")
+                        os.remove(file_path / (rss_.name + ".json"))
                     except BaseException as e:
                         logger.info(e)
                     await Rssdel.send('订阅 ' + rss_name + ' 删除成功！')
@@ -69,6 +74,5 @@ async def handle_RssAdd(bot: Bot, event: Event, state: dict):
                 RWlist.writeRss(list_rss)
                 await Rssdel.send('删除 ' + str(flag) + ' 条订阅！')
     except BaseException as e:
-        #logger.info(e)
+        # logger.info(e)
         await Rssdel.send('你还没有任何订阅！\n关于插件：http://ii1.fun/7byIVb')
-

--- a/src/plugins/ELF_RSS2/plugins/show_all.py
+++ b/src/plugins/ELF_RSS2/plugins/show_all.py
@@ -1,10 +1,11 @@
 from RSSHUB import RWlist
 from nonebot import on_command, permission
-from nonebot.rule import to_me
 from nonebot.adapters.cqhttp import Bot, Event
+from nonebot.rule import to_me
 
+RssShowAll = on_command('showall', aliases={'showall', 'seeall'}, rule=to_me(), priority=5,
+                        permission=permission.SUPERUSER)
 
-RssShowAll = on_command('showall', aliases={'showall','seeall'}, rule=to_me(), priority=5,permission=permission.SUPERUSER)
 
 @RssShowAll.handle()
 async def handle_first_receive(bot: Bot, event: Event, state: dict):
@@ -26,7 +27,7 @@ async def handle_first_receive(bot: Bot, event: Event, state: dict):
                 if str(group_id) in str(rss_.group_id):
                     msg = msg + '名称：' + rss_.name + '\n订阅地址：' + rss_.url + '\n\n'
                     flag += 1
-                    if(flag%5==0):
+                    if (flag % 5 == 0):
                         await RssShowAll.send(msg)
                         msg = ''
             if flag <= 0:
@@ -41,8 +42,8 @@ async def handle_first_receive(bot: Bot, event: Event, state: dict):
             for rss_ in list_rss:
                 msg = msg + '名称：' + rss_.name + '\n订阅地址：' + rss_.url + '\n\n'
                 flag += 1
-                #每条信息展示 5 条订阅
-                if(flag%5==0):
+                # 每条信息展示 5 条订阅
+                if (flag % 5 == 0):
                     await RssShowAll.send(msg)
                     msg = ''
             if flag <= 0:
@@ -53,4 +54,3 @@ async def handle_first_receive(bot: Bot, event: Event, state: dict):
             await RssShowAll.send('还没有任何订阅！')
     else:
         await RssShowAll.send('你没有权限进行此操作！\n关于插件：http://ii1.fun/7byIVb')
-

--- a/src/plugins/ELF_RSS2/plugins/show_dy.py
+++ b/src/plugins/ELF_RSS2/plugins/show_dy.py
@@ -1,13 +1,15 @@
+from pathlib import Path
+
 from RSSHUB import RWlist
-from nonebot import permission
 from nonebot import on_command
-from nonebot.rule import to_me
+from nonebot import permission
 from nonebot.adapters.cqhttp import Bot, Event
+from nonebot.rule import to_me
 
 # 存储目录
-file_path = './data/'
+file_path = Path.cwd() / 'data'
 
-RssShow = on_command('show', aliases={'showdy','lookdy'}, rule=to_me(), priority=5,permission=permission.SUPERUSER)
+RssShow = on_command('show', aliases={'showdy', 'lookdy'}, rule=to_me(), priority=5, permission=permission.SUPERUSER)
 
 
 @RssShow.handle()
@@ -33,8 +35,9 @@ async def handle_RssAdd(bot: Bot, event: Event, state: dict):
             if rss_.name == rss_name or rss_.url == rss_name:
                 await RssShow.send(
                     '名称：' + rss_.name + '\n订阅地址：' + rss_.url + '\n订阅QQ：' + str(rss_.user_id) + '\n订阅群：' + str(
-                        rss_.group_id) + '\n更新频率：' + str(rss_.time) + '分钟/次\n代理：' + str(rss_.img_proxy) + '\n第三方：' + str(rss_.notrsshub)
-                            + '\n翻译：' + str(rss_.translation) + '\n仅标题：' + str(rss_.only_title) + '\n仅图片：' + str(rss_.only_pic))
+                        rss_.group_id) + '\n更新频率：' + str(rss_.time) + '分钟/次\n代理：' + str(
+                        rss_.img_proxy) + '\n第三方：' + str(rss_.notrsshub)
+                    + '\n翻译：' + str(rss_.translation) + '\n仅标题：' + str(rss_.only_title) + '\n仅图片：' + str(rss_.only_pic))
                 flag = flag + 1
         if flag <= 0:
             await RssShow.send('没有找到 ' + rss_name + ' 的订阅哟！')

--- a/src/plugins/ELF_RSS2/plugins/start.py
+++ b/src/plugins/ELF_RSS2/plugins/start.py
@@ -1,9 +1,13 @@
-import nonebot
-from bot import config
-from RSSHUB import del_cache as DC, rsstrigger as RT, RWlist
 import asyncio
+
+import nonebot
+from RSSHUB import del_cache as DC, rsstrigger as RT, RWlist
 from nonebot import on_metaevent, logger
 from nonebot.adapters.cqhttp import Bot, Event
+
+from bot import config
+
+
 async def start():
     bot, = nonebot.get_bots().values()
     try:
@@ -15,17 +19,22 @@ async def start():
         rss_list = RWlist.readRss()  # 读取list
         for rss in rss_list:
             RT.rss_trigger(rss.time, rss)  # 创建检查更新任务
-        await bot.send_msg(message_type='private', user_id=str(list(config.superusers)[0]), message='ELF_RSS 订阅器启动成功！\nVERSION: {}'.format(config.version))
+        await bot.send_msg(message_type='private', user_id=str(list(config.superusers)[0]),
+                           message='ELF_RSS 订阅器启动成功！\nVERSION: {}'.format(config.version))
         logger.info('ELF_RSS 订阅器启动成功！')
     except Exception as e:
-        await bot.send_msg(message_type='private', user_id=str(list(config.superusers)[0]), message='第一次启动，你还没有订阅，记得添加哟！\nVERSION: {}'.format(config.version))
+        await bot.send_msg(message_type='private', user_id=str(list(config.superusers)[0]),
+                           message='第一次启动，你还没有订阅，记得添加哟！\nVERSION: {}'.format(config.version))
         logger.info('第一次启动，你还没有订阅，记得添加哟！')
+        e.with_traceback()
+
 
 def startfun():
     loop = asyncio.get_event_loop()
     loop.run_until_complete(start())
 
-#region 启动问好
+
+# region 启动问好
 def check_first_connect(bot: Bot, event: Event, state: dict) -> bool:
     if event.sub_type == 'connect':
         return True

